Makefile

```diff
@@ -61,12 +61,15 @@ watch-unit:
 watch-compile:
 	$(BIN_DIR)/tsc --watch  --noEmit --skipLibCheck
 
+oathkeeper:
+	oathkeeper serve -c dev/ory/oathkeeper.yml --sqa-opt-out
+
 e2e:
+	# need oathkeeper running in another terminal
 	yarn build && \
 	yarn test:e2e
 
 e2e-in-ci:
-	docker compose -f docker-compose.yml up integration-deps -d && \
```

Comment:
are you sure about this? probably will break in concourse

Reply:
we don't need this anymore I think because we now have hardcoded variable environment. we no longer need to run the script to populate macaroon and tls.

as you still think that can break concourse (?) I will make a separate PR out of it

Reply:
relevant commit: https://github.com/GaloyMoney/galoy/commit/040eaafbd80a9a392ad9a34eb66b9d22712c9ccb

Reply:
https://github.com/GaloyMoney/galoy/pull/1668